pkg/gardenlet/controller/seed/seed/reconciler_reconcile.go

```diff
@@ -1041,19 +1042,28 @@ func (r *Reconciler) runReconcileSeedFlow(
 		return flow.Errors(err)
 	}
 
-	// Patching fluent-bit service with the network policy annotations
-	fluentBitService := &corev1.Service{}
-	if err := seedClient.Get(ctx, kubernetesutils.Key(v1beta1constants.GardenNamespace, v1beta1constants.DaemonsetNameFluentBit), fluentBitService); err != nil {
-		return err
-	}
-
-	fluentBitServicePatch := client.MergeFrom(fluentBitService.DeepCopy())
-	fluentBitService.ObjectMeta.Annotations[v1beta1constants.AnnotationNetworkPolicyFromPolicyPodLabelSelector] = v1beta1constants.NetworkPolicyAllScrapeTargets
-	fluentBitService.ObjectMeta.Annotations[v1beta1constants.AnnotationNetworkPolicyFromPolicyAllowedPorts] = "[{\"port\":\"2020\",\"protocol\":\"TCP\"},{\"port\":\"2021\",\"protocol\":\"TCP\"}]"
+	if loggingEnabled {
+		// Waiting for fluent-bit service to be created
```

Comment:
Do we have plans to address this in the upstream? If yes, we can link issue, add TODO?

Reply:
Yes, there is already merged PR with this and we are waiting for a release now